### ISSUE TITLE
Plugin is not working

### ISSUE THREAD

**Issue by MadridHappyPeople** [dudaster/ele-custom-skin#950]
I have bought rith now the anual package but it is not working. Please return the money back and see issue attached. Thank you

**Comment by nndesignac** [comment]
I have the same error. Probably since the latest Elementor update. Urgently ask for a quick solution. The error only affects "Ele Custom Skin Pro", as soon as you disable it, the page works again. Previous emergency solution: downgrade Elementor to version "3.25.10".

**Comment by chrisdt** [comment]
Same error on multiple sites. Rolling Elementor to version "3.25.10" works for all our sites as mentioned above.

**Comment by dogsfored** [comment]
Yes, I have just experienced the same thing.  Updating to Elementor Version 3.26.0 broke all the pages where Ele Custom Skin Pro loops were loading. Disabled the Pro plugin and resolved the issue, just without the PRO features. Any fix??

**Comment by 1000camels** [comment]
See this for a fix: https://github.com/dudaster/ele-custom-skin/issues/949
